### PR TITLE
[COD-36] Can register user when password confirmation is not passed backend (fixed)

### PR DESCRIPTION
**Pull request by LorDDeadWolf**
Fixed an issue when user could be created with unexisted fields
https://legolas.atlassian.net/browse/COD-36?atlOrigin=eyJpIjoiZmMwN2YyOTNiNDZkNDc1YmIzZTFiMTA4MzY5ZDQ1YWYiLCJwIjoiaiJ9
Снимок экрана 2023-02-16 121837

### CODE REVIEW

**Comment by Set27** [comment]
Check rubocop issued

File: backend/spec/models/user_spec.rb [hunk 1]
```diff
@@ -3,12 +3,73 @@
...
-  end
+  subject(:user) { described_class.create(user_params) }
+
+  let(:user_params) { { last_name:, first_name:, email:, password:, password_confirmation:, role: } }
+  let(:last_name) { 'Lastname' }
+  let(:first_name) { 'Firstname' }
+  let(:email) { 'test@mail.ru' }
+  let(:password) { 'testtest' }
+  let(:password_confirmation) { 'testtest' }
+  let(:role) { 'student' }
```
Comment (SilaEwGenii) [inline backend/spec/models/user_spec.rb:14]: @npupko вопрос здесь

**Review by SilaEwGenii — APPROVED** [review]
LGTM

**Review by npupko — APPROVED** [review]
LGTM

**Pull request merged**